src/scancode/cli.py

```diff
@@ -467,6 +469,26 @@ def scan_event(item):
     return files_count, cached_scan.iterate(scans, root_dir)
 
 
+def shorten_filename(filename='', max_length=25, number_of_dots=3):
+    """
+    Return the original filename if it is short enough
```

Comment:
What about rather:
`Return a truncated `filename` usable for fixed width display if `filename` is longer than `max_length`.`
Also why do we need a default for `filename=''`? Calling this function without args doe not make sense. It should just error out. Nothing special needed. `number of dots` should not be an arg. This is not something I would need to ever modify.